backend/src/services/product.service.ts

```diff
@@ -3,6 +3,8 @@ import { User } from '../models/user.model';
 import { Request } from 'express';
 import { ProductImageAttributes, ProductImage } from '../models/productImage.model';
 import { upload, MulterRequest } from '../middlewares/imageUpload';
+import {Op} from "sequelize";
+import {userInfo} from "os";
```

Comment:
Is this really needed?

Reply:
Op yes, 
userInfo I didn't even notice. Seems to be outdated code, that wasn't deleted.